pulsar-io/data-generator/pom.xml

```diff
@@ -49,6 +49,18 @@
             <artifactId>jfairy</artifactId>
             <version>0.5.9</version>
         </dependency>
+        <dependency>
+            <groupId>com.google.inject</groupId>
+            <artifactId>guice</artifactId>
+            <version>${guice.version}</version>
+            <scope>provided</scope>
+        </dependency>
+        <dependency>
+            <groupId>com.google.inject.extensions</groupId>
+            <artifactId>guice-assistedinject</artifactId>
+            <version>${guice.version}</version>
+            <scope>provided</scope>
+        </dependency>
```

Comment:
Would it be useful to add these to the root parent pom.xml's dependencyManagement / dependencies so that the correct versions get picked in all locations?

Reply:
I intended to comment on #14300